src/sagemaker/analytics.py

```diff
@@ -463,6 +465,8 @@ def __init__(
         self._sort_order = sort_order
         self._metric_names = metric_names
         self._parameter_names = parameter_names
+        self._input_artifact_names = input_artifact_names
+        self._output_artifact_name = output_artifact_names
```

Comment:
You might want to add comments for these 2 arguments in the above section as well.

Reply:
added